packages/models/src/serialize.js

```diff
@@ -60,7 +60,7 @@ export function serializeFilter(filter) {
     const filter = declutter[k];
     if (Array.isArray(v) && v.length !== 0) {
       memo[k] = v;
-    } else if (filter && filter.default !== v) {
+    } else if (filter && (options.explicit || filter.default !== v)) {
```

Comment:
What makes this necessary? In my mind there should be no functional difference between serialized states where the `explicit` flag differs, the full state should be captured either way.

Are the defaults dynamic?

Reply:
Yeah, [there is one spot ](https://github.com/getappmap/appmap-js/blob/main/packages/components/src/pages/VsCodeExtension.vue#L791)where we change the default value. But I can make it so that the defaults are not dynamic. Should I do that, or should we merge this PR?

Reply:
In that case I think this is fine. I'll approve the PR.